sdcm/cluster.py

```diff
@@ -797,6 +803,11 @@ def is_enterprise(self):
             return "scylla-enterprise" in result or "No matching items found" not in result
         return "scylla-enterprise" in self.remoter.sudo("apt-cache search scylla-enterprise", ignore_status=True).stdout
 
+    @cached_property
```

Comment:
on rolling upgrade we might need to clear this cache, or just leave it without cache.